src/servers/shared_memory_manager.cc

```diff
@@ -172,6 +172,16 @@ SharedMemoryManager::RegisterSystemSharedMemory(
             .c_str());
   }
 
+  err = CloseSharedMemoryRegion(shm_fd);
```

Comment:
What if MapSharedMemory above fails... then shm_fd is not closed.

Reply:
Done